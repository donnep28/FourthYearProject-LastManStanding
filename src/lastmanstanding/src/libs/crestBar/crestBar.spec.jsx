import React from 'react';
import {shallow} from 'enzyme';
import CrestBar from './crestBar';

describe('crestBar', () => {
  describe('Snapshot', () => {
    it('renders', () => {
      const component = shallow(
          <CrestBar/>,
      );
      expect(component).toMatchSnapshot();
    });
  });
});