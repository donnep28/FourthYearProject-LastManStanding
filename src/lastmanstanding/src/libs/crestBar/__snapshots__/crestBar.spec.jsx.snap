// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`crestBar Snapshot renders 1`] = `
<Crests
  crests={Array []}
  screenWidth={null}
/>
`;
